central/cluster/datastore/telemetry.go

```diff
@@ -14,6 +17,10 @@ import (
 
 const securedClusterClient = "Secured Cluster"
 
+// clusterIdentityCache keeps the last reported properties of the connected
+// secured clusters.
+var clusterIdentityCache = expirable.NewLRU(100, func(key string, value map[string]any) {}, 24*time.Hour)
```

Comment:
```suggestion
var clusterIdentityCache = expirable.NewLRU[string, any](100, nil, 24*time.Hour)
```

Reply:
Maybe we can use a hash of map instead of storing full map 

https://pkg.go.dev/hash/maphash